legion/engine/rendering/pipeline/default/stages/meshrenderstage.cpp

```diff
@@ -18,8 +18,8 @@ namespace legion::rendering
         static id_type lightsId = nameHash("light buffer");
         static id_type lightCountId = nameHash("light count");
         static id_type matricesId = nameHash("model matrix buffer");
-        static id_type sceneColorId = nameHash("scene color");
-        static id_type sceneDepthId = nameHash("scene depth");
+        //static id_type sceneColorId = nameHash("scene color history");
+        //static id_type sceneDepthId = nameHash("scene depth history");
```

Comment:
Perhaps these can fully removed? or perhaps be added back in if it was for debug testing

Reply:
nope but i left comment saying why